src/main/java/com/smartling/marketo/sdk/rest/MarketoEmailRestClient.java

```diff
@@ -34,15 +34,20 @@
     }
 
     @Override
-    public Email loadEmailById(int id) throws MarketoApiException {
-        List<Email> execute = httpCommandExecutor.execute(new LoadEmailById(id));
-        if (execute != null && !execute.isEmpty()) {
-            return execute.get(0);
+    public Email loadEmailById(int id, Status status) throws MarketoApiException {
```

Comment:
I would rename this method as far as we added a new parameter

Reply:
As for me, it's regular method overloading.

Reply:
1 endpoint - 1 method name
Almost all Marketo endpoints have status as a parameter, by default, it is APPROVED
I just introduced a more correct signature to be able to load draft as well, for testing and troubleshooting purposes

Reply:
got it, thanks